doc/manual/variables-and-scoping.rst

```diff
@@ -49,16 +49,20 @@ Julia uses `lexical scoping <https://en.wikipedia.org/wiki/Scope_%28computer_sci
 meaning that a function's scope does not inherit from its caller's
 scope, but from the scope in which the function was defined.
 For example, in the following code the ``x`` inside ``foo`` refers
-to the ``x`` in the global scope of its module ``Bar``::
+to the ``x`` in the global scope of its module ``Bar``:
```

Comment:
> I'm horrible at RST – is the double colon unnecessary here because the doctest directive takes care of it?

That's my understanding from playing with it. Furthermore adding `::` after `Bar` here causes "`.. doctest::`" to show up in the `.html` file.